### ISSUE TITLE
Gauge Highcharts plotbands are not accessible via keyboard navigation 

### ISSUE THREAD

**Issue by skubalQB** [highcharts/highcharts#15471]
#### Expected behavior
Expecting the plotbands to be accessible on keyboard navigation and show the data labels just like mouse over event

#### Actual behaviour
Keyboard navigation for plotband doesn't work, neither the data is being shown.

#### Live demo with steps to reproduce
https://jsfiddle.net/Delfin_/vpryjzet/1/

#### Product version
9.0.1

#### Affected browser(s)
Tested on mozilla and Chrome

**Comment by KacperMadej** [comment]
Thank you for reporting the problem.

It is possible to add an Accessibility component - docs: https://api.highcharts.com/class-reference/Highcharts.AccessibilityComponent#AccessibilityComponent - to provide a wide range of a11y support (focusable, screen reader info, etc.).

@skubalQB - For a solid gauge chart (like in your demo), you could use points in the solid gauge series as they are focusable, has full a11y support, and have a build-in tooltip. https://www.highcharts.com/demo/gauge-solid

@oysteinmoseng - Should this issue stay open as a feature request for an a11y module support for plotBand?

**Comment by oysteinmoseng** [comment]
Yes, this particular use case seems to be using plot bands a bit differently than what they normally are. Normal plot bands are not generally interactive, and as such should not be part of the keyboard navigation. A custom component or using points is probably the best solution for this case.

We should still expose plot lines and plot bands to screen readers, though, so we can leave the issue open I think @KacperMadej.
